<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgSettings</class>
 <widget class="QDialog" name="CQDlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QRadioButton" name="userUnitRadians">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>31</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Radians</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="userUnitDegrees">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>50</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Degrees</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>User interface units</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="userUnitSeconds">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Seconds</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="userUnitMilliseconds">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>31</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Milliseconds</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="userUnitMillimeters">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>31</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Millimeters</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="userUnitMeters">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Meters</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>361</width>
     <height>66</height>
    </rect>
   </property>
   <property name="title">
    <string>Object manipulation</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Translation step size</string>
    </property>
   </widget>
   <widget class="QComboBox" name="translationStepSize">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation step size</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rotationStepSize">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Vertices/triangles verification settings</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="removeIdenticalVerticesCheckbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Remove identical vertices, tolerance:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="removeIdenticalVerticesTolerance">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="removeIdenticalTriangles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Remove identical triangles</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ignoreTriangleWinding">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Ignore triangle winding</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Various settings</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="antialiasing">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Antialiasing</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="worldReference">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Display world reference</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="boundingBoxDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Display bounding boxes of selected objects</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="undoRedo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Undo / redo enabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hideConsole">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Hide console window</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqAutoSave">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Auto-save enabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqTransparentDlgs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Transparent dialogs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqExternalScriptEditor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>External script editor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqScriptEditor">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="qqSelectScriptEditor">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqWorkThreads">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Enable work threads</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqThreadedRendering">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Threaded rendering</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqAdjustOpenGl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust OpenGL settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>361</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Additional parameters can be adjusted in file system/usrset.txt</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>userUnitRadians</zorder>
  <zorder>userUnitDegrees</zorder>
  <zorder>userUnitSeconds</zorder>
  <zorder>userUnitMilliseconds</zorder>
  <zorder>userUnitMillimeters</zorder>
  <zorder>userUnitMeters</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>label_3</zorder>
 </widget>
 <tabstops>
  <tabstop>userUnitRadians</tabstop>
  <tabstop>userUnitDegrees</tabstop>
  <tabstop>userUnitMillimeters</tabstop>
  <tabstop>userUnitMeters</tabstop>
  <tabstop>userUnitMilliseconds</tabstop>
  <tabstop>userUnitSeconds</tabstop>
  <tabstop>translationStepSize</tabstop>
  <tabstop>rotationStepSize</tabstop>
  <tabstop>removeIdenticalVerticesCheckbox</tabstop>
  <tabstop>removeIdenticalVerticesTolerance</tabstop>
  <tabstop>removeIdenticalTriangles</tabstop>
  <tabstop>ignoreTriangleWinding</tabstop>
  <tabstop>antialiasing</tabstop>
  <tabstop>worldReference</tabstop>
  <tabstop>boundingBoxDisplay</tabstop>
  <tabstop>undoRedo</tabstop>
  <tabstop>hideConsole</tabstop>
  <tabstop>qqTransparentDlgs</tabstop>
  <tabstop>qqAutoSave</tabstop>
  <tabstop>qqWorkThreads</tabstop>
  <tabstop>qqThreadedRendering</tabstop>
  <tabstop>qqExternalScriptEditor</tabstop>
  <tabstop>qqScriptEditor</tabstop>
  <tabstop>qqSelectScriptEditor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
